dex/src/main/scala/com/wavesplatform/dex/AddressActor.scala

```diff
@@ -118,6 +118,16 @@ class AddressActor(owner: Address,
         context.actorOf(BatchOrderCancelActor.props(toCancelIds.toSet, self, sender, batchCancelTimeout))
       }
 
+    case command: Command.CancelOrders =>
+      val allActiveOrderIds = getActiveLimitOrders(None).map(_.order.id()).toSet
```

Comment:
`val allActiveOrderIds = getActiveLimitOrders(None).map(_.id).toSet`

Reply:
Unfortunately this isn't available on a stable branch